main.py

```diff
@@ -0,0 +1,52 @@
+import csv
+
+
+# def read_data_for_new_task(filepath):
+#     with open(filepath, 'r') as csv_file:
+#         reader = csv.reader(csv_file)
+#         for row in reader:
+#             yield row
+
+class ReadData:
+    def __init__(self, csv_file_path):
+        while True:
+            self.count = 0
+            self.filepath = csv_file_path
+            messege = input('Введите Enter для вывода следующего значения или End для выхода из цикла: ')
```

Comment:
If you create a class ReadData, it should just read data and doesn't do anything else